app/views/contacts/_needs-table.html.erb

```diff
@@ -23,6 +23,14 @@
                 </td>
                 <td><%= n.assigned %></td>
                 <td><%= n.created_at&.strftime("%-d %B %Y") %></td>
+                <td>
+                  <%= n.start_on&.strftime("%-d %B %Y") %>
+
+                  <%# is this a 'future' need %>
+                  <% if !n.start_on.nil? && n.start_on > DateTime.now %>
+                     <strong>Padlock</strong>
```

Comment:
In case we don't get round to adding the svg today, maybe write the word "Future" or something that would make sense for the time being